src/cli/query.js

```diff
@@ -1,22 +1,15 @@
-const chalk = require('chalk')
 const ora = require('ora')
-const columnify = require('columnify')
-const dateFormat = require('date-fns/format')
 
-const query = require('../api/query')
+const { query } = require('../api/graphql')
```

Comment:
I imagine we'll need to change to use this import in a bunch of other places?

Reply:
Good call, I first thought it wasn't worth it but we've now got shared error handling logic.